services/scabbard/cli/src/main.rs

```diff
@@ -150,6 +150,11 @@ fn run() -> Result<(), CliError> {
                                 .takes_value(true)
                                 .possible_values(&["human", "csv"])
                                 .default_value("human"),
+                            Arg::with_name("key")
```

Comment:
We should probably use a different name than `key` here. The `--key` argument is used by some commands to specify the key for signing transactions; it's very possible that that's a different key than the one used to authenticate with the REST API.

Reply:
Like Peter said, I was under the impression that the key used for signing transactions would be the same key used for signing json web tokens

Reply:
Didn't realize we were assuming the keys were the same. Sounds good.